public/wp-content/themes/newsup-child/single-anime.php

```diff
@@ -7,6 +7,7 @@
 <!-- =========================
      Page Content Section
 ============================== -->
+single-anime.php
```

Comment:
Why is this a commit. This is a bug.

Reply:
i was testing something, ill remove it before i request to merge